src/client/app/access-checker/admin-cache-entries.component.ts

```diff
@@ -38,8 +38,8 @@ export class AdminCacheEntriesComponent {
 		this.alertService.clearAllAlerts();
 
 		this.pagingOpts = new PagingOptions();
-		this.pagingOpts.sortField = this.sortOpts['key'].sortField;
-		this.pagingOpts.sortDir = this.sortOpts['key'].sortDir;
+		this.pagingOpts.sortField = this.sortOpts.key.sortField;
```

Comment:
Updated the PR so style guide files live in config/styleguide - However, I left the gulp glob so it is able to pull from the other typescript files. It will ignore comments that aren't in the livingcss format, so we can have in-line comments next to affected concepts. For example, if we want to define language (ie: "What is a stream? What is a group?")